airflow/models/baseoperator.py

```diff
@@ -1021,8 +1039,8 @@ def xcom_pull(
     @cached_property
     def extra_links(self) -> Iterable[str]:
         """@property: extra links for the task. """
-        return list(set(self.operator_extra_link_dict.keys())
-                    .union(self.global_operator_extra_link_dict.keys()))
+        return sorted(list(set(self.operator_extra_link_dict.keys())
```

Comment:
```suggestion
        return sorted(set(self.operator_extra_link_dict.keys())
```

Reply:
We might not need the set either - dicts can't have duplicate keys

```suggestion
        return sorted(self.operator_extra_link_dict.keys())
```

Reply:
I am going to remove all the Extra link commits and changes to baseoperator once https://github.com/apache/airflow/pull/6302 is merged.